include/boost/geometry/algorithms/simplify.hpp

```diff
@@ -208,9 +208,13 @@ struct simplify_ring
             // Verify area around closing point, if that can be simplified,
             // start/end are modified and a corresponding slice will be used
             // for simplification
+
+            // Take only 10% of the simplify distance, to avoid to aggressive
+            // behaviour at closing points (it is known as a "open problem")
+
             // TODO: for open polygons, implementation should be modified
             simplify_closure_inspector::get_start_end(start, end, ring,
-                    max_distance, strategy);
+                    max_distance / 10.0, strategy);
```

Comment:
Where does value 10% come from? Is it a well known "open problem" or it comes from some testcase?